src/barebones_mpc/model/inv_pendulum_model.py

```diff
@@ -11,23 +22,35 @@ def __init__(self, time_step, number_samples, sample_length, cart_mass, pendulum
         u: force applied to the cart
 
         """
-        self.input_dimension = 1
-        self.state_dimension = 4
+        super().__init__(
+            time_step=time_step,
+            number_samples=number_samples,
+            sample_length=sample_length,
+            state_dimension=state_dimension,
+        )
+
         self.time_step = time_step
         self.number_samples = number_samples
         self.sample_length = sample_length
+
+        self.input_dimension = 1
+        self.state_dimension = state_dimension
+
         self.cart_mass = cart_mass
         self.pendulum_mass = pendulum_mass
         self.epsilon = pendulum_mass / (cart_mass + pendulum_mass)
         print(self.epsilon)
 
-        self.state_transition_matrix = np.array([[0, 1, 0, 0],
-                                                 [0, 0, -self.epsilon, 0],
-                                                 [0, 0, 0, 1],
-                                                 [0, 0, 1, 0]])
-        self.input_transition = np.array([0, 1, 0, -1]).reshape((4 , 1))
+        self.state_transition_matrix = np.array([[0, 1, 0, 0], [0, 0, -self.epsilon, 0], [0, 0, 0, 1], [0, 0, 1, 0]])
```

Comment:
I prefer when matrices as arrays are divided by row to properly see each row.

Reply:
The trick to prevent black from formating those line is to either insert a comment like `# fmt: skip` before the array or to add any comment in the array after a `],`